gen3_util/files/middleware.py

```diff
@@ -0,0 +1,35 @@
+
+import pathlib
+
+from gen3_util.config import Config
+from gen3_util.repo import CLIOutput
+from gen3_util.common import to_metadata_dict
+from gen3_util.meta.skeleton import transform_manifest_to_indexd_keys
+from gen3_util.files.lister import ls
+
+
+def files_ls_wr(config: Config, object_id: str, project_id: str, specimen: str, patient: str, observation: str, task: str, md5: str, is_metadata: bool, is_snapshot: bool, long: bool):
```

Comment:
@matthewpeterkort Could you include a doc string and comment?

Reply:
Addressed in https://github.com/ACED-IDP/gen3_util/pull/42/commits/8b101bed29dbad86dfd16f83a2fb6f0a5523c85d